src/GraphQL/Extensions/TypeExtensions.cs

```diff
@@ -164,9 +164,9 @@ public static Type GetGraphTypeFromType(this Type type, bool isNullable = false,
                 }
                 else if (mode == TypeMappingMode.OutputType)
                 {
-                    var outputAttr = type.GetCustomAttribute<OutputTypeAttribute>();
-                    if (outputAttr != null)
-                        graphType = outputAttr.OutputType;
+                    //var outputAttr = type.GetCustomAttribute<OutputTypeAttribute>();
```

Comment:
But also we have a problem when reading those attributes - should specify generic type.

Reply:
Yeah, the resolution was obvious.